catalog/templates/column.html

```diff
@@ -192,16 +277,36 @@
                "normalization": "predenominated"}]'
             , 1, 1) meta
           FROM data)
-          SELECT id, (data->0->>'value')::{{ col.type }} as per_sq_km
+          SELECT id, (data->0->>'value')::{{ col.type }} as raw_count
```

Comment:
Uhm, I'm not sure if it's available in the RSTs, but the optimal solution would be to use the same column name we autogenerate for Builder, i.e: the `suggested_name` column of the GetMeta result. 

If not, we can probably add it to the query to generate rsts, or replicate this logic in python:
https://github.com/CartoDB/observatory-extension/blob/develop/src/pg/sql/41_observatory_augmentation.sql#L249-L257